public/app/plugins/panel/geomap/style/markers.ts

```diff
@@ -39,13 +40,31 @@ export function getFillColor(cfg: StyleConfigValues) {
   return undefined;
 }
 
+const textLabel = (cfg: StyleConfigValues) => {
+  const fontFamily = config.theme2.typography.fontFamily;
+  const textConfig = { 
+    ...defaultStyleConfig.textConfig,
+    ...cfg.textConfig,
+   }
+  return new Text({
+    text: cfg.text ?? '?',
+    fill: new Fill({ color: cfg.color ?? defaultStyleConfig.color.fixed }),
+    font: `normal ${fontSize}px ${fontFamily}`,
```

Comment:
`fontSize` probably needs to be pulled from the cfg object (`fontSize` property). This is throwing another TS error due to `fontSize` not being defined